notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -555,7 +555,7 @@ export default {
           { name: 'links', items: [ 'simpleLink','InsertOptions'] },
         ],
         format_tags: 'p;h1;h2;h3',
-        autoGrow_minHeight: self.noteFormContentHeight,
+        autoGrow_minHeight: true,
```

Comment:
can you please check this change, in fact this attribute must be of type `Number`, see https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-autoGrow_minHeight